src/Dto/AcceptAffectation.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Dto;
+
+use App\Entity\Affectation;
+use App\Entity\User;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class AcceptAffectation
+{
+    private ?Affectation $affectation = null;
```

Comment:
Est ce qu'il manquerait pas une assert not null pour l'affectation ?
ça sécurise cette ligne dans le formType 
`        $choicesAgents = $builder->getData()->getAffectation()->getPartner()->getUsers();
`